preliz/distributions/distributions.py

```diff
@@ -204,11 +193,11 @@ def mode(self):
         raise NotImplementedError
 
     def std(self):
-        """Standard deviation."""
+        """Calculate Standard Deviation."""
```

Comment:
Standard deviation.

Reply:
We don't use calculate for the other properties/moments

Reply:
This is done, added `noqa: D401` as I was getting a ruff error here